advocacy_docs/partner_docs/QuestFoglightforDatabases/04-ConfiguringQuestFoglightforDatabases.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: 'Configuring Quest Foglight for Databases'
+description: 'Walkthrough on configuring Quest Foglight for Databases'
+---
+
+Implementing Quest Foglight for Databases with EDB Postgres Advanced Server and/or EDB Postgres Extended Server requires the following components:
+
+- EDB Postgres Advanced Server and/or EDB Postgres Extended Server
+- Quest Foglight Software
+
+### Prerequisites
+
+- A running EDB Postgres Advanced Server or EDB Postgres Extended Server instance
+- Quest Foglight Software installed
+- Login credentials for your EDB Advanced Server or EDB Postgres Extended instance and Quest Foglight interface
+
+### Configuring Quest Foglight for EDB Postgres Advanced Server and EDB Postgres Extended Server
+
+The following steps show how to configure Quest Foglight for Databases to connect to EDB Postgres Advanced servers and EDB Postgres Extended Servers:
+
+1. Log on to the Quest Foglight console
+
+   ![Quest Login](Images/QuestLoginFrame.png)
+
+2. Once on the home page, ensure you are in ‘Expert View’ by navigating to the toolbar on the left and switching from ‘Standard’ to ‘Expert’.  Then navigate to the Databases section by clicking on 'Databases' in the menu:
+
+   ![Quest Databases Menu](Images/QuestDatabasesScreen.png)
+
+3. Select PostgreSQL as the database server and click on 'Monitor' ( + icon) to launch the Agent Installer Wizard:
+
+   ![Quest Monitor Agent Type](Images/QuestMonitorAgentType.png)
+
+4. Enter the desired agent name and click Next
+
+**EDB Postgres Advanced Server example:**
```

Comment:
This looks like a heading they way it sticks out.  I am thinking it might look better under the image and be right justified.  Can you try this out so we can see what it looks like?

Also can you review the images and make sure you think they are helpful.  The one under step 2 and step 3 I think could be a closer up image. or maybe use the image you have with a zoomed in image over it.

Reply:
I've worked on some formatting and would like to see what you think if that's the direction we'd like to go with the image text.

I have also fixed the image issues you were discussing and think they look more detailed now.